DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -42,12 +42,11 @@ public enum SessionReplay {
         guard configuration.replaySampleRate > 0 else {
             return
         }
+        let resources = ResourcesFeature(core: core, configuration: configuration)
+        try core.register(feature: resources)
 
         let sessionReplay = try SessionReplayFeature(core: core, configuration: configuration)
         try core.register(feature: sessionReplay)
-
-        let resources = ResourcesFeature(core: core, configuration: configuration)
-        try core.register(feature: resources)
```

Comment:
Interestingly enough - this order matters. Since we use resources data store in session replay recorder it requires resources feature to be registered before.

Raises a question if it would make sense to add new function that registers array of features.